Framework/Built_In_Automation/Performance_Testing/BuiltInFunctions.py

```diff
@@ -0,0 +1,225 @@
+          assign locust user | performance action | existing locust variable name
+    Returns:
+        "passed" if success.
+        "zeuz_failed" otherwise.
+    """
+
+    sModuleInfo = inspect.currentframe().f_code.co_name + " : " + MODULE_NAME
+
+    try:
+        locust_var = losust_var_name = user_name = user_type = wait_time = host = None
+        try:
+            for left, mid, right in data_set:
+                left = left.strip().lower()
+                if mid.strip().lower() == "input parameter":
+                    if "user name" == left:
+                        user_name = right.strip()
+                    if "type" == left:
+                        user_type = right.strip().lower()
+                        user_type = 'HttpUser' if user_type == 'httpuser' else "User" if user_type == "user" else None
+                    elif "wait_time" == left:
+                        wait_time = right.strip().lower()
+                    elif "host" == left:
+                        host = right.strip().lower()
+                elif mid.strip().lower() == "action":
+                    if "assign locust user" == left:
+                        losust_var_name = right.strip()
+            if None in [losust_var_name,user_name,user_type,wait_time,host]: 
+                CommonUtil.ExecLog(sModuleInfo,  f"dataset is inaccurate", 3)
+                return "zeuz_failed"
+                
+            locust_var = sr.Get_Shared_Variables(losust_var_name,log=False)
+            locust_var['users'][user_name] = {'type':user_type,'wait_time' : wait_time,'host':host,'tasks':[],'user_task_sets':[]}
+        except:
+            CommonUtil.ExecLog(sModuleInfo, "Failed to parse data.", 1)
+            traceback.print_exc()
+            return "zeuz_failed"
+        sr.Set_Shared_Variables(losust_var_name, locust_var)
+        return "passed"
+    except:
+        return CommonUtil.Exception_Handler(sys.exc_info())
+
+
+@logger
+def assign_locust_taskset(data_set):
+    """
+    Add Locust user configuration to the existing locust variable
+    This action provides that data to creates a user class into locust file
+    Args:
+        data_set:
+          wait_time          | input parameter    | between(int,int)
+          host               | input parameter    | url of host
+          name               | input parameter    | user/class name Case sensitive
+          type               | input parameter    | type of user/class
+          assign locust user | performance action | existing locust variable name
+    Returns:
+        "passed" if success.
+        "zeuz_failed" otherwise.
+    """
+
+    sModuleInfo = inspect.currentframe().f_code.co_name + " : " + MODULE_NAME
+
+    try:
+        locust_var = losust_var_name = taskset_name = user_names =  None
+        try:
+            for left, mid, right in data_set:
+                left = left.strip().lower()
+                if mid.strip().lower() == "input parameter":
```

Comment:
this condition should not be mandatory. If someone writes "element parameter" I think we should allow them